repoqa/search_needle_function.py

```diff
@@ -374,6 +385,8 @@ def evaluate_model(
     eval_ignore_comments: bool = False,  # ignore comments during score computation
     trust_remote_code: bool = False,
     attn_implementation=None,
+    is_embedding: bool = False,
+    embedding_context_chunk_size: int = 30
```

Comment:
is this # of chunks or # tokens for each chunk? better make it more explicit :)
also i think specifying the size /#token of each chunk might be more intuitive

Reply:
Currently it's lines of code, will change the variable name